Sealnote/src/main/java/com/twistedplane/sealnote/data/Note.java

```diff
@@ -93,7 +93,7 @@ private void readFromParcel(Parcel inParcel) {
         mArchived = inParcel.readInt() > 0;
         mDeleted = inParcel.readInt() > 0;
         mType = Type.valueOf(inParcel.readString());
-        mNote = NoteContent.fromString(mType, inParcel.readString());
+        mNote = NoteContent.fromString(mType, note);
```

Comment:
Why did you remove the `inParcel.readString()` and replaces with note? `note` is used just once here, so original code seems fine.